.pryrc

```diff
@@ -6,6 +6,6 @@ if defined?(PryByebug)
 end
 
 # Hit Enter to repeat last command
-Pry::Commands.command /^$/, 'repeat last command' do
+Pry::Commands.command (/^$/), 'repeat last command' do
```

Comment:
I... don't think this does what you think it does????

Reply:
I think you mean `Pry::Comments.command(/^$/, 'repeat last command') do`, to be clear

Reply:
yeah.. I think you are right.

Reply:
well technically what I did is "right" (I think my stupidity beat rubocop). But anyways, changed it to be more correct.